### DISCUSSION TITLE
State Layout in CommunityToolkit MAUI

### DISCUSSION THREAD

**Discussion by Kremed** [CommunityToolkit/Maui#677] [Q&A]
Is it available now in this version of Xamarin MAUI CommunityToolkit? I've searched for a long time on the internet but nothing about it.
If it is in progress, where can I see the scheduling of upcoming tasks for the Xamarin MAUI CommunityToolkit?

I tried this but it didn't work !!, It's related to Xamarin.Forms  :

`xmlns:xct="http://xamarin.com/schemas/2020/toolkit"`
`<StackLayout  toolkit:StateLayout.CurrentState="" toolkit:StateLayout.AnimateStateChanges="False"/>`

**Comment by VladislavAntonyuk** [comment]
you can use MauiCompat version, which contains StateLayout.
It will be available soon in MCT (after merging this PR: https://github.com/CommunityToolkit/Maui/pull/673)

**Comment by nicjay** [comment]
Working on it! It's shaping up nicely, usage will be similar to XCT.

**Comment by CliffAgius** [comment]
Closed and answered
